superset/translations/es/LC_MESSAGES/messages.po

```diff
@@ -789,28 +795,28 @@ msgstr "La fuente no existe"
 
 #: superset/common/query_object.py:301
 msgid "`operation` property of post processing object undefined"
-msgstr ""
+msgstr "La propiedad `operation` del objeto de post procesamiento no esta definida"
 
 #: superset/common/query_object.py:305
 #, python-format
 msgid "Unsupported post processing operation: %(operation)s"
-msgstr ""
+msgstr "Operación de post procesamiento no soportada: %(operation)s"
 
 #: superset/connectors/druid/models.py:240
 msgid "Adding new datasource [{}]"
-msgstr "Añadiendo [{}] como nueva fuente"
+msgstr "Añadiendo [{}] como nueva fuente de datos"
 
 #: superset/connectors/druid/models.py:243
 msgid "Refreshing datasource [{}]"
-msgstr "Actualizando la fuente [{}]"
+msgstr "Actualizando la fuente de datos [{}]"
 
 #: superset/connectors/druid/models.py:1054
 msgid "Metric(s) {} must be aggregations."
-msgstr "la(s) métrica(s) {} tiene que ser agregaciones"
+msgstr "La(s) métrica(s) {} tiene que ser agregaciones"
```

Comment:
🤔   "agregados" is meant as a noun but in this context can be misread as a verb:  It could be read as "_...must be aggregated_" while it really means "_...must be aggregations_").  The ambiguity is very subtle and harmless, though...

Maybe:  "**La(s) métrica(s) {} deben ser agregado(s).**" helps get rid of ambiguity by forcing _agregado(s)_ to be read as a noun

Reply:
Yeah, I feel you there. It's subtle and AFAIK there's no good solution as an "aggregate" as a noun is not something naturally use in the Spanish language. I was hoping that the difference in gender (femenine for metrics, masculine for aggregate) would signal that this is not a verb, but rather it's a noun. But I can see how, to some people, this may come across as a verb or as a grammar mistake.

Some alternatives I can think of are not as accurate or a bit too verbose for my taste, but may help disambiguate:

"La(s) métrica(s) {} deben ser agregadas" (not as accurate - this litterally reads as "you must aggregate the metrics)
"La(s) métrica(s) {} deben pertenecer a un agregado" (too verbose and somewhat inaccurate - reads as "metrics must belong to an aggregate")

All in all I was more inclined to use "agregados" as a noun for "aggregate".